fk/luae_fs/src/test/java/se/inera/intyg/common/luae_fs/v1/model/converter/InternalToCertificateTest.java

```diff
@@ -71,4 +73,10 @@ void shallIncludeMetadata() {
         final var actualCertificate = internalToCertificate.convert(internalCertificate, textProvider);
         assertNotNull(actualCertificate.getMetadata(), "Shall contain metadata");
     }
+
+    @Test
+    void shallIncludeCategoryGrundForMU() {
+        final var actualCertificate = internalToCertificate.convert(internalCertificate, textProvider);
+        assertEquals(0, actualCertificate.getData().get(GRUNDFORMU_CATEGORY_ID).getIndex());
+    }
```

Comment:
Seems like there is a test missing here for QuestionUtlatandeBaseratPa?

Reply:
Added